docs/oid-info.md

```diff
@@ -58,11 +95,90 @@ the git ref that the workflow run was based upon.
 This specifies the username identity in the OtherName Subject Alternative Name, as
 defined by [RFC5280 4.2.1.6](https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.6).
 
+
+### 1.3.6.1.4.1.57264.1.8 | Source Repository URI
+
+Should include a fully qualified source repository URL that the build was based
+on. For example: `https://example.com/owner/repository`.
+
+### 1.3.6.1.4.1.57264.1.9 | Source Repository Digest
+
+An immutable reference to a specific version of the source code that the build
+was based upon. For example: `abc123` git commit SHA.
+
+### 1.3.6.1.4.1.57264.1.10 | Source Repository Ref
+
+The source ref that the build run was based upon. For example: `refs/head/main`.
+
+### 1.3.6.1.4.1.57264.1.11 | Source Repository Identifier
+
+The immutable identifier for the source repository the workflow was based upon. For example: `1234` if using a primary key.
+
+### 1.3.6.1.4.1.57264.1.12 | Source Repository Owner URI
+
+Fully qualified URL for the owner of the source repository that the build was based
+on, e.g. `https://example.com/owner`
+
+### 1.3.6.1.4.1.57264.1.13 | Source Repository Owner Identifier
+
+The immutable identifier for the owner of the source repository that the workflow was based upon. For example: `5678` if using a primary key.
+
+### 1.3.6.1.4.1.57264.1.14 | Build Config URI
+
+A reference to the initiating build instructions. For example: `https://example.com/owner/repository/build-config.yml`.
+
+### 1.3.6.1.4.1.57264.1.15 | Build Config Digest
+
+An immutable reference to the specific version of the top-level build
+instructions. For example: `abc123` git commit SHA.
+
+### 1.3.6.1.4.1.57264.1.16 | Build Signer URI
+
+Reference to specific build instructions that are responsible for signing. Can be the same as Build Config URI. Could be a reusable workflow ref in GitHub Actions or a Circle CI Orb name. For example: `https://github.com/slsa-framework/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@v1.4.0`.
+
+### 1.3.6.1.4.1.57264.1.17 | Build Signer Digest
+
+An immutable reference to the specific version of the build instructions that is responsible for signing. For example: `abc123` git commit SHA.
+
+### 1.3.6.1.4.1.57264.1.18 | Build Trigger
+
+The event or action that triggered the build. For example: `push`.
+
+### 1.3.6.1.4.1.57264.1.19 | Run Invocation URI
+
+An immutable identifier that can uniquely identify the build execution. For GitHub Actions this needs to include both the run id and attempt. For example: `https://github.com/example/repository/actions/runs/1536140711/attempts/1`.
```

Comment:
Should we omit "For GHA this needs to include ...", since that is documented below already?